TAG-self-review-questionnaire.md

```diff
@@ -0,0 +1,132 @@
+This storage partition should be cleared when a user wishes to clear storage for that origin.
+
+### 2.7. Do the features in your specification expose information about the underlying platform to origins?
+
+No, this API does not expose any information about the underlying platform.
+
+### 2.8. Does this specification allow an origin to send data to the underlying platform?
+
+No, this API does not send information to the underlying platform.
+
+### 2.9. Do features in this specification enable access to device sensors?
+
+No, this API does not provide access to device sensors.
+
+### 2.10. Do features in this specification enable new script execution/loading mechanisms?
+
+No, this API does not provide any new capabilities to load or execute scripts.
+
+### 2.11. Do features in this specification allow an origin to access other devices?
+
+No, this API does not provide access to any other devices.
+
+### 2.12. Do features in this specification allow an origin some measure of control over a user agent’s native UI?
+
+No, this API does not provide control over a user agent's native UI.
+The user agent may choose to indicate in its native UI when this feature is being used.
+
+### 2.13. What temporary identifiers do the features in this specification create or expose to the web?
+
+This API does not create or expose any temporary identifiers to the web.
+Third parties using this API may use the storage partition to persist identifiers provided by other APIs, but this could be done with any storage mechanism.
+
+### 2.14. How does this specification distinguish between behavior in first-party and third-party contexts?
+
+This API is only available to third-party contexts that are embedded in a Related Website Set.
+
+### 2.15. How do the features in this specification work in the context of a browser’s Private Browsing or Incognito mode?
+
+Related Website Sets is not available in Private Browsing or Incognito mode, so the RWP API is not available as well.
+
+### 2.16. Does this specification have both "Security Considerations" and "Privacy Considerations" sections?
+
+[Yes.](https://github.com/explainers-by-googlers/related-website-partition-api#privacy-and-security-considerations)
+
+### 2.17. Do features in your specification enable origins to downgrade default security protections?
+
+No, this feature does not provide any capability to opt out of any security protections.
+
+### 2.18. What happens when a document that uses your feature is kept alive in BFCache (instead of getting destroyed) after navigation, and potentially gets reused on future navigations back to the document?
+
+This feature gives third parties access to a persistent storage partition for use within an RWS.
+The storage partition will be available to the document before and after navigation, but the document will have to invoke the RWP API again to access it.
```

Comment:
Huh, it's available in non-fully-active documents?

Reply:
Is the RWS grant preserved if you go back to a page that previously had it granted?

Reply:
Yeah the permission itself is granted for 30 days. Maybe I misunderstood the response - "before and after" means it's not available in BFCache, right?

Reply:
Ah, yeah this should probably just say 'after'.

Reply:
Done.